workflow/create_experiment.py

```diff
@@ -79,7 +79,7 @@ def input_args():
 
     if 'skip_ci_on_hosts' in testconf:
         host = Host()
-        if host.machine.lower() in testconf.skip_ci_on_hosts.lower():
+        if host.machine.lower() in [ machine.lower() for machine in testconf.skip_ci_on_hosts ]:
```

Comment:
```suggestion
        if host.machine.lower() in [machine.lower() for machine in testconf.skip_ci_on_hosts]:
```

Reply:
Sorry, this was my fault in the previous suggestion.

Reply:
fixed